cmd/tendermint/commands/compact.go

```diff
@@ -0,0 +1,55 @@
+package commands
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+func MakeCompactDBCommand(cfg *config.Config, logger log.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "experimental-compact-db",
+		Short: "force compacts the tendermint storage engine (only GoLevelDB supported)",
+		Long: `
+This is a temporary utility command that performs a force compaction on the state 
```

Comment:
If this is temporary, can we avoid having it live in the command-line tool entirely? It seems like this could be a separate program that the operator can run if this affects them.

If we keep wanting to do this sort of thing longer-term, I wonder if we shouldn't make an `experimental` subsection of the CLI (e.g., `tendermint experimental screw-up-my-settings-because-i-like-it-that-way --force`).

Reply:
How I've been seeing the tendermint cli (and this can be of course subject to change) is as a utility tool predominantly for node operators to assist them in their operations. In this regard, I'm happy to bloat out the binary with various commands so long as the cli remains intuitive to use and structured. 

I think there's merit in the point that operators running out of process nodes perhaps don't want all this extra stuff hanging on to their binary

Reply:
I think the view that the `tendermint` binary doesn't matter makes sense if all nodes are expected to be wrapped in the SDK. And I guess that's the most common case. But I don't think that means we should just say "anything goes" for folks who want to run a non-SDK chain.

Perhaps what we need is a second designated `tmutils` tool, where the kitchen sink can go.